decision-process/README.md

```diff
@@ -0,0 +1,134 @@
+---
+creation_date: 2023-10-10
+issues:
+  - https://github.com/giantswarm/giantswarm/issues/24661
+owners:
+  - https://github.com/orgs/giantswarm/teams/sig-architecture
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+  - https://github.com/orgs/giantswarm/teams/team-team
+state: review
+---
+
+# RFC and decision making process
+
+This RFC-of-RFCs aims to expand on the current RFC process by introducing an explicit, lightweight decision making process.
+
+We use the terms "RFC" and "decision" quite interchangeably. Think of a _decision_ as exactly what the word means, and an _RFC_ as its written-down version as Markdown file.
+
+At the moment, the concept of [ADRs](https://intranet.giantswarm.io/docs/product/architecture-specs-adrs/adr/) and [PDRs](https://intranet.giantswarm.io/docs/product/pdr/) are not covered here and their processes do not change yet. We may want to amend this later.
+
+Goals of the process:
+
+- Quick progress on RFCs and decision making
+- Relevant groups have seen and approved the decision
+- More structured RFC format to add rendering and further automation
+- Better overview by rendering RFCs
+- Replace the [rfc repository's README](https://github.com/giantswarm/rfc/blob/b3f4d2af6ff8965e542baae7d0e8fadb9163f356/README.md)
+- Giant Swarm folks using RFCs more readily and happily because they see that the process is working
+
+## RFC file structure
+
+To allow for a decision making process without removing the current free-form nature of RFCs, some structured information is proposed.
+
+The following changes will be introduced:
+
+- All RFCs will have a YAML header with structured information that can be parsed and used e.g. for rendering the RFCs in our [handbook](https://handbook.giantswarm.io/)
+- The YAML header must contain the following fields in this order:
+
+  - `creation_date: YYYY-MM-DD`
+  - `issues` containing links to GitHub issues. Can be null or an empty list. The issues can be the origin of the RFC or where the work for implementing the decision takes place.
```

Comment:
When would this be acceptable to be null?

Reply:
i think i'm okay with this - my thinking is that i want all (important) rfcs to have some tracking information, but maybe that's too heavyweight

let's go with this for now